### PR TITLE
chore(argocd): update notification notifiers' comment

### PR DESCRIPTION
**Pull request by yu-croco**
This PR is revert of https://github.com/argoproj/argo-helm/pull/1220 ( because the configuration already existed 🙇 ) and is to improve comment.

Note on DCO:

If the DCO action in the integration test fails, one or more of your commits are not signed off. Please click on the *Details* link next to the DCO action for instructions on how to resolve this.

Checklist:

* [x] I have bumped the chart version according to [versioning](https://github.com/argoproj/argo-helm/blob/master/CONTRIBUTING.md#versioning)
* [x] I have updated the documentation according to [documentation](https://github.com/argoproj/argo-helm/blob/master/CONTRIBUTING.md#documentation)
* [x] I have updated the chart changelog with all the changes that come with this pull request according to [changelog](https://github.com/argoproj/argo-helm/blob/master/CONTRIBUTING.md#changelog).
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/tree/master/community#contributing-to-argo).
* [x] My build is green ([troubleshooting builds](https://argoproj.github.io/argo-cd/developer-guide/ci/)).

Changes are automatically published when merged to `master`. They are not published on branches.

### CODE REVIEW

File: charts/argo-cd/Chart.yaml [hunk 1]
```diff
@@ -21,4 +21,4 @@ dependencies:
     condition: redis-ha.enabled
 annotations:
   artifacthub.io/changes: |
-    - "[Feat]: Add ability to set Notification Services"
+    - "[chore]: update comment to clarify usecase"
```
Comment (mkilchhofer) [inline charts/argo-cd/Chart.yaml:24]:
```suggestion
    - "[Changed]: update comment to clarify usecase"
```
We use these templates here:
https://github.com/argoproj/argo-helm/blob/master/CONTRIBUTING.md#changelog

Comment (yu-croco) [reply charts/argo-cd/Chart.yaml]: done.

**Review by mkilchhofer — APPROVED** [review]
💯 lgtm. thanks for fixing it 🙏

**Comment by yu-croco** [comment]
Thank you for reviewing! and sorry for the inconvenience. 🙇

**Review by mbevc1 — APPROVED** [review]

**Comment by mbevc1** [comment]
Sorry for missing it first time around, YAML :zany_face:

**Pull request merged**